packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -295,6 +297,12 @@ export const LivenessCameraModule = (
 
   const shouldShowCenteredLoader = isInitCamera || isInitWebsocket;
 
+  const shouldAllowCameraSelection =
+    isStartView &&
+    selectableDevices &&
+    selectableDevices.length > 1 &&
+    (!isMobileScreen || (allowBackCamera && isFaceMovementChallenge));
```

Comment:
Makes sense to me, only allow camera selection on mobile if `allowBackCamera` and is face movement challenge

Reply:
+1, this is a very readable conditional, nice work!